ynput/OpenPype
# Maya: Tweak renderlayer behavior to adhere more to variant changes

## Changelog Description

Tweak renderlayer behavior to adhere more to variant changes + allow more than once creation to create renderlayers.

The enum option now defaults to the previously existing behavior, only ever creating the Rendersetup Layer if it *does not exist yet*. However, now it does adhere to the variant name so that the created layer is of the chosen variant..

## Additional info

Based on discord discussion [here](https://discord.com/channels/517362899170230292/517382145552154634/1183745985080463440)

## Testing notes:

1. Launch Maya
2. Create render instances + layers, publish, and submit to deadline, etc.
3. make sure renderlayer name changes accordingly update the instances' variants, etc.
